prefect_airbyte/client.py

```diff
@@ -12,38 +12,36 @@ class AirbyteClient:
     """
     Client class used to call API endpoints on an Airbyte server.
 
-    This client assumes that you're using an OSS Airbyte server which does not require
-    an API key to access its a API.
+    This client currently supports username/password authentication as set in `auth`.
 
     For more info, see the [Airbyte docs](https://docs.airbyte.io/api-documentation).
 
     Attributes:
+        airbyte_base_url str: Base API endpoint URL for Airbyte.
+        auth: Username and password for Airbyte API.
         logger: A logger instance used by the client to log messages related to
             API calls.
-        airbyte_base_url str: Base API endpoint URL for Airbyte.
         timeout: The number of seconds to wait before an API call times out.
+        client: An `httpx.AsyncClient` instance used to make API calls.
     """
 
     def __init__(
         self,
         logger: logging.Logger,
         airbyte_base_url: str = "http://localhost:8000/api/v1",
+        auth: Tuple[str, str] = ("airbyte", "password"),
         timeout: int = 5,
     ):
+        self._closed = False
+        self._started = False
+
         self.airbyte_base_url = airbyte_base_url
+        self.auth = auth
         self.logger = logger
         self.timeout = timeout
-
-    async def _establish_session(self) -> httpx.AsyncClient:
-        """
-        Checks health of the Airbyte server and establishes a session.
-
-        Returns:
-            Session used to communicate with the Airbyte API.
-        """
-        client = httpx.AsyncClient(timeout=self.timeout)
-        await self.check_health_status(client)
-        return client
+        self.client = httpx.AsyncClient(
+            base_url=self.airbyte_base_url, auth=self.auth, timeout=self.timeout
+        )
```

Comment:
Should this be private?

Reply:
I agree that this should probably be private.
```suggestion
        self._client = httpx.AsyncClient(
            base_url=self.airbyte_base_url, auth=self.auth, timeout=self.timeout
        )
```